### ISSUE TITLE
Paladin

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#11]
Issues and notes on the Paladin

UT2004 Draw  - 1.0
UT3    Draw  - 1.35
UT3Port Draw - 1.0 / 1.15 estimate by visual test or 1.20 which feels like it's too long but 1.15 in the front feels slightly not high enough
                          1.17 looks like it might work
Both UT3 port and Stock UT2004 Paladin don't seem to cast any light ahead

  - [Priority]
  - [Try 1.17 Drawscale]
  - [Wheels have no collision on port but do on UT3]  
  - [Front plating has no collision but does on UT3]  
  - [Turret has no collsion on port but does on UT3]  
  - [Back plating has collision but it's very close so your arm clips it,  on UT3 the collision is quite far all the way across the back where you don't even touch the back wheels]  
  - [Secondary]  
  - [Shield just hovers in air without any energy beams from the turret holding it suspended on the port]  
  - [Turret turns from the front to the back in 2 seconds on UT3 and nearly 4 seconds on port]  
  - [You can almost flip the port Paladin over with a hoverboard lol :D , not in UT3, it doesn't even budge but at least the sound effect is exact]  
  - [Port is missing front left lights and in fact doesn't seem to have any lights]  
  - [Shield gun section of the turret is illuminated kind of on UT3 but not on the port]  
  - [Extra]  
  - [Primary fire might be slightly faster on the port, clicking both the mouse and my controller at the same time it seemed like the PC port fired a few milliseconds quicker]  
  - [Shield deactivates if the bottom of it goes below the ground on UT3 but not on the port]  
  - [Shield is shock colored and not team colored on UT3]

**Comment by Unre-Alex** [comment]
Turret's size is too small, changing it to 1.2 looks more accurate.

**Comment by HellDragon-HK** [comment]
Hey Alex, you wouldn't know the console command for sizing up the turret would you, I'd like to give it a shot and see what I get since I can see both games at the same time it's easy to compare....is that true on your side as well if so then let's do our best to catch everything yeah?

**Comment by HellDragon-HK** [comment]
Impact results with Paladin

Paladin Impact with Manta
-------UT3 Results--------
--Front Hit   - Fair knockback, easily pushed
--Side Hit    - Fair knockback, easily pushed
--Back Hit    - Fair knockback, easily pushed
--Diagnol Hit - Little knockback (no impact sound), easily pushed
------UT2004 Results------ 
--Front Hit   - Fair knockback and some spin, easily pushed like on ice
--Side Hit    - Fair knockback, easily pushed like on ice
--Back Hit    - Fair knockback, easily pushed like on ice
--Diagnol Hit - Fair knockback, easily pushed like on ice

Paladin Impact with Scorpion
-------UT3 Results--------
--Front Hit   - Little knockback and Scorpion went vertical, cannot be pushed but can be lifted up until vertical where it then is thrown away (kind of felt like a Sumo competetion is the best way I can describe it)
--Side Hit    - Little knockback and Scorpion flipped on it's side, cannot be pushed but does eventually roll over and the Paladin drives over it
--Back Hit    - Little knockback and Scorpion was lifted off back wheels, cannot be pushed but is eventually flipped over
--Diagnol Hit - Little knockback, cannot be pushed but is easily flipped
------UT2004 Results------ 
--Front Hit   - Little knockback, easily pushed but doesn't appear to flip
--Side Hit    - Little knockback and Scorpion almost flipped, flips instead of gets pushed
--Back Hit    - Little knockback, easily pushed
--Diagnol Hit - Little knockback, can be turned but not really pushed
Extra Note: UT3 Paladin had impact sound each hit and UT2004 Paladin had no impact sound but I have heard the impact sound in UT2004 when hitting the Paladin with other vehicles so it isn't completely missing

Paladin Impact with Hellbender
-------UT3 Results--------
--Front Hit   - Little knockback, cannot be pushed but flipped and acted like a ramp launching the Paladin into the air over it which was pretty neat and you could probably use that trick to launch the Paladin over low walls and possibly the barricades even
--Side Hit    - Little knockback and Hellbender was lifted off ground, pushing almost causes ramp trick again but instead this time Hellbender went completely upside down and Paladin sat on top of it
--Back Hit    - Little knockback and Hellbender lifted off back wheels, ramp trick done in reverse when pushing, Hellbender went slanted on Paladin and was launched just like the Paladin was earlier
--Diagnol Hit - Little knockback, cannot be pushed and eventually gets lifted and launched
------UT2004 Results------ 
--Front Hit   - Little knockback, easily pushed (no launching here :( )
--Side Hit    - Little knockback and Hellbender was lifted slightly off the ground, easily pushed and tries to flip but doesn't
--Back Hit    - Little knockback, easily pushed no flipping
--Diagnol Hit - Little or no knockback, turns instead of gets pushed
Extra Note: On UT3 I successfully launched the Hellbender over the barricade and the Paladin with it's front wheels still in the air came down on top of the barricade and also went over so new tactic learned from bump testing :D only thing is whoever is in the Hellbender is going to be sick as it vertically flipped all the way around at least once while it was in the air before landing (I also did this all in the 1st attempt so it's probably fairly easy to do)

Paladin Impact with Stealthbender
-------UT3 Results--------
--Front Hit   - unable to find map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - pending
--Side Hit    - pending
--Back Hit    - pending
--Diagnol Hit - pending

Paladin Impact with Raptor
-------UT3 Results--------
--Front Hit   - Little knockback, can be pushed
--Side Hit    - Little knockback, easily pushed
--Back Hit    - Little knockback, easily pushed
--Diagnol Hit - Little knockback, easily pushed
------UT2004 Results------ 
--Front Hit   - Fair knockback, pushed like on ice
--Side Hit    - Fair knockback, pushed like on ice
--Back Hit    - Fair knockback, pushed like on ice
--Diagnol Hit - Fair knockback, pushed and turned like on ice
Extra Note: In UT2004 Raptor damaged and Paladin was not

Paladin Impact with Cicada
-------UT3 Results--------
--Front Hit   - Unable to find map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - Little knockback, pushed like on ice
--Side Hit    - Fair knockback and Paladin drives over Cicada, cannot be pushed but can be turned until rotor section hits Paladin which causes Paladin to flip on it's side
--Back Hit    - Little to fair knockback, can be pushed but not as easily
--Diagnol Hit - Same as side hit including Paladin getting flipped
Extra Note: In UT2004 Cicada took as lot of damage and Paladin took very slight damage

Paladin Impact with Hellfire SPMA
-------UT3 Results--------
--Front Hit   - cannot find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - Little to no knockback, easily pushed
--Side Hit    - Little to no knockback and slight flip attempt, can be pushed and almost flipped
--Back Hit    - Little to no knockback, easily pushed and SPMA wheel plating moving with wheels again
--Diagnol Hit - Little to no knockback and SPMA slightly lifted off ground, pushing attempt caused SPMA to lift on top of Paladin where it was easily pushed and flipped

Paladin Impact with Paladin
-------UT3 Results--------
--Front Hit   - No knockback and both Paladins lifted off front wheels slightly, cannot be pushed or flipped
--Side Hit    - No knockback and other Paladin slightly lifted off ground while driving Paladin tried to drive over it, cannot be pushed or flipped it seems
--Back Hit    - No knockback and other Paladin slightly lifted off back wheels and driving Paladin slightly lifted off front wheels, cannot be pushed or flipped
--Diagnol Hit - No knockback and both Paladins lifted off front wheels slightly, cannot be pushed or flipped
------UT2004 Results------ 
--Front Hit   - No knockback, can be pushed until the Paladins go on top of each other and then they are both launched with the other Paladin (which went over mine) is launched straight up in the air sideways and my Paladin was launched in the direction I was moving sideways and also noticed caused game to stutter
--Side Hit    - No knockback, slight tap causes Paladin to flip and trying to push any further like this resulted in both Paladins being launched rolling horizonatlly into the air in opposite directions
--Back Hit    - No knockback, can be slowly pushed without bizare behavior seen previously
--Diagnol Hit - No knockback, slightest tap caused other Paladin to bottom up which it came back down on my Paladin and both were flipped upside and launched and game ejected me from Paladin.....most exciting impact testing I've had so far
Extra Note: Both Paladins damaged, probably from the launching behavior

Paladin Impact with Goliath
-------UT3 Results--------
--Front Hit   - No knockback, pushing managed to make both vehicles go vertical and got stuck in an arch pattern
--Side Hit    - No knockback and slight lift up on Goliath, cannot be pushed or flipped
--Back Hit    - No knockback, can be slowly pushed
--Diagnol Hit - No knockback, can be slowly pushed
------UT2004 Results------ 
--Front Hit   - Little to no knockback, can be pushed
--Side Hit    - Little knockback, can be pushed
--Back Hit    - Little knockback, can be pushed
--Diagnol Hit - Little knockback, can be pushed and easily turned

Paladin Impact with Leviathan
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - Little knockback, can be slowly pushed
--Side Hit    - Little to fair knockback, cannot be pushed
--Back Hit    - Little knockback, can be pushed
--Diagnol Hit - Little knockback, can be pushed

Paladin Impact with Viper
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - Fair to medium knockback, easily pushed and turned like on ice
--Side Hit    - Fair knockback, easily pushed and turned like on ice
--Back Hit    - Fair knockback, easily pushed and turned like on ice
--Diagnol Hit - Fair knockback, easily pushed and turned like on ice

Paladin Impact with Scavenger
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - pending
--Side Hit    - pending
--Back Hit    - pending
--Diagnol Hit - pending

Paladin Impact with Nightshade
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - Fair knockback, easily pushed like on ice
--Side Hit    - Fair knockback with almost 90 degree spin, easily pushed and turned like on ice
--Back Hit    - Fair knockback, easily pushed like on ice
--Diagnol Hit - Fair knockback, easily pushed and turned like on ice

Paladin Impact with Fury
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - pending
--Side Hit    - pending
--Back Hit    - pending
--Diagnol Hit - pending

Paladin Impact with Nemesis
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - pending
--Side Hit    - pending
--Back Hit    - pending
--Diagnol Hit - pending

Paladin Impact with Darkwalker
-------UT3 Results--------
--Front Hit   - unable to find a map that has both
--Side Hit    - 
--Back Hit    - 
--Diagnol Hit - 
------UT2004 Results------ 
--Front Hit   - pending
--Side Hit    - pending
--Back Hit    - pending
--Diagnol Hit - pending

**Comment by Unre-Alex** [comment]
@HellDragon-HK The fastest way is to open UT3VH_Paladin_Anims.ukx in editor and on the right mesh->scale->change all 3 to desired one

**Comment by HellDragon-HK** [comment]
Ok I'll try that, luckily I already know just enough about the editor to be able to navigate it

**Comment by HellDragon-HK** [comment]
I still can't quite tell if it's really needed....I'm going to forgo waiting for Emerald's say on if I can install your textures, you said you added the lights and well with those missing it's kind of hard to determine so I need those.

**Comment by Unre-Alex** [comment]
Just replace them, they won't effect any functionality.

**Comment by HellDragon-HK** [comment]
Alright then and in case you missed the edit I don't see a Fury texture, was it left out by accident?

**Comment by Unre-Alex** [comment]
Blue texture doesn't work with the current build, so I didn't add it.

**Comment by HellDragon-HK** [comment]
Ok so not a bug just thought you might want to know

**Comment by Unre-Alex** [comment]
The class for Fury (UT3Fury.uc) is missing the reference to a blue skin

**Comment by HellDragon-HK** [comment]
Hadn't noticed that actually from in-game, I'll add it to the Fury's list

**Comment by HellDragon-HK** [comment]
@Unre-Alex Big problem, I installed your textures and now I don't have UT3 vehicles spawning, just the stock UT2004 vehicles....any ideas?

**Comment by Unre-Alex** [comment]
replace viper and nightshade with old textures and see again

**Comment by HellDragon-HK** [comment]
Ok good thing I was hellbent on not replacing anything and renamed them instead

**Comment by Unre-Alex** [comment]
I think it might be because of the viper, I modified it with Beta 1 to have the right projectiles

**Comment by HellDragon-HK** [comment]
nope didn't fix the problem

**Comment by Unre-Alex** [comment]
did you add UT3VehicleEffects.utx?

**Comment by HellDragon-HK** [comment]
Yeah and renamed the original, I'm using beta4 release but GreatEmerald had me download a new archive so maybe it's not compatible?

**Comment by Unre-Alex** [comment]
there was no UT3VehicleEffects.utx before, I use the new build too...

**Comment by HellDragon-HK** [comment]
I don't think the archive I have is released but only GE can confirm or disprove that....I'll look at the textures in the editor and see if there isn't a name difference somewhere

**Comment by Unre-Alex** [comment]
Try maybe with Manta only, if it doesn't work then you have to have some other build

**Comment by HellDragon-HK** [comment]
Would running UCC Make reveal the problem you think and I'll try Manta first

**Comment by HellDragon-HK** [comment]
@Unre-Alex That worked (Manta tex only right now) so I'll try each file one at a time until I find the problem

**Comment by Unre-Alex** [comment]
Redownload the pack again, you must have downloaded it right after I posted it, but later I updated the pack.

**Comment by HellDragon-HK** [comment]
I did grab it right after yeah, ok give me a second, btw did you notice the Stealthbender tex is missing?

**Comment by Unre-Alex** [comment]
yes, it's not in its class.

**Comment by HellDragon-HK** [comment]
so is that another bug like the Fury that needs to be posted you mean? If so can you tell me exactly what it is so it can be listed?

**Comment by Unre-Alex** [comment]
There's no reference to its texture at all, just for the hellbender, but there should be references to 2 textures.

**Comment by HellDragon-HK** [comment]
Ok so list as Missing texture reference in class then?  Btw do you get the bizarre behavior I do with the Stealthbender in UT2004?

**Comment by Unre-Alex** [comment]
yes, it's a complete mess, I do not touch it :)

**Comment by HellDragon-HK** [comment]
Lol yep that's my philosophy right now as well

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Ok so the turret looks right now except how tall it is and the wheels also have this issue the body thanks to your lights being added to it looks like it might be the right height  
btw will you be able to make those work on 64 bit?

**Comment by Unre-Alex** [comment]
Interesting, the Scorpion is the only one that I used blurring for its light, so I guess I would have to make that for all. Sure, but once GE will make necris vehicles useable.

**Comment by HellDragon-HK** [comment]
So I'm going to have to stick with 32 bit for now, ah boy it has some issues for me but I'll try
Playing around with the turret here and so far the Z scale of 1.5 is looking more accurate but I looking harder it's still not tall enough, long enough, and it also from what I can see in UT3 is the same length sideways as the Paladin's body

**Comment by Unre-Alex** [comment]
is this for the body or turret?

**Comment by HellDragon-HK** [comment]
Turret, also just found out in UT3 I was on ground slightly higher and after moving I think I'm sure that the UT2004 Paladin's body needs to be taller

**Comment by Unre-Alex** [comment]
hold on I can not look at both games the same time, I let you know

**Comment by HellDragon-HK** [comment]
Sure I'll hold btw I think you need a bit more red in the front lights although it's quite close

**Comment by Unre-Alex** [comment]
yea, I compared screenshots and it looks fine with 1.1 body

**Comment by HellDragon-HK** [comment]
What about the turret?
Also I set the scale of the body back to 1.0 and it's still floating in the air, so are it's wheels, and still has a collision circle around it that makes it impossible to even shoot the thing (rockets explode well before they reach it now)

**Comment by Unre-Alex** [comment]
didn't you change the Z translation?

**Comment by HellDragon-HK** [comment]
No because I don't know what to change it to, this is new territory for me man

**Comment by Unre-Alex** [comment]
replace it with the old file?

**Comment by HellDragon-HK** [comment]
I can do that yeah but I'd like to actually be able to fix it and start really helping Emerald & Gping....if only I could learn how to do this :S

**Comment by Unre-Alex** [comment]
Knock yourself out ,I'm not doing things the hard way

**Comment by HellDragon-HK** [comment]
I'm trying sigh, got an opinion on the turret?

**Comment by Unre-Alex** [comment]
thre's something slightly off about that 1.6

**Comment by HellDragon-HK** [comment]
I know and I think it's that the turret actually needs to be literally moved over to the right a little

**Comment by Unre-Alex** [comment]
If there's no light on some vehicle that means it need corona or emitter.

**Comment by HellDragon-HK** [comment]
I mean on UT3 it isn't there unless it's a brake light which I can't see while driving

**Comment by Unre-Alex** [comment]
So it's just a matter of adding bMakeBrakeLights

**Comment by HellDragon-HK** [comment]
To be clear I'm saying with your textures you added a light detail that isn't there in UT3 from what I can see if you added it to show where brakelights should go then ok I get that, I can't see in UT3 that part while driving and if it is brakelights then it only shows when driving

**Comment by Unre-Alex** [comment]
Got you, than it's similar with the Scorpion and Hellbender. GE would have to code to show texture only on brake, and the light textures would have to be split.

**Comment by HellDragon-HK** [comment]
Oh ok I see what you mean, make it invisible and combine with brakelight so it has the UT3 appearance, question is does the UT3 Paladin in UT3 actually have brakelights

**Comment by Unre-Alex** [comment]
Just checked and no, not ever they light up.  Also, the shield is waaaay smaller in UT3.

**Comment by HellDragon-HK** [comment]
Same result here, man bots don't want to use the Paladin in Downtown it seems took forever to get somebody to drive it ok I'll list the shield, looks like it's about the same size as the Paladin's body or turret

btw you do understand I'm saying you've got lights that are not supposed to be there now right it doesn't have any lights in back   not trying to upset you just trying ti get things as accurate as possible friend

**Comment by Unre-Alex** [comment]
Yes I said I got you

**Comment by HellDragon-HK** [comment]
Wow you were not even close to kidding about the shield how did I miss that

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
If you don't mind I'd like to compare your lights job with what I see in UT3 and suggest changed I'm looking at the Manta now, and the inside controls are not bright they are a dull dark red, the middle bar at head height isn't bad but could use a touch more red in it as could the front and back other than that looks like you got all of the spots :D  I also am trying to keep in mind my TV colors may differ as the red hud elements are really bright red with the orb symbol being more of a hot pink   

Also where would you like these comments to go, I'd like to have them for the vehicles they are meant for but going where you actually linked the pack would be good too

**Comment by Unre-Alex** [comment]
This is the only place where I shared the textures (out of respect).

You can gather notes about the lights, and once your done, maybe send it to my email. Keep in mind I'm not going to work on them unless GE will make the Necris vehicles functioning, or he might wanna work on the lights himself.

As for UT3 characters, I was working on them for a while, but only finished the Liandri and Necris. They were never meant to be released (out of respect for the creator), but If you want them give me your email. I might have to check all the dependencies first, as I was fixing team skins not fully working for the Liandri, so maybe tomorrow.

**Comment by HellDragon-HK** [comment]
I've started adding them as new check marks to each vehicle at the very bottom so check there, a lot of it is just needing some subtle detail like the missing glow for the Manta.

I don't think UrPackage (at least that's who I recall did that pack) would mind it being released as an add-on after all you still need his model pack but I understand.

my e-mail is draconicraptor@Yahoo.com I'll delete this soon though as I don't want to be spammed so let me know when you've got it wrote down or something please.

**Comment by Unre-Alex** [comment]
you can delete it now

**Comment by HellDragon-HK** [comment]
ok thanks for doing that so quickly

**Comment by Unre-Alex** [comment]
I'm sure he can make it better, hell I could make it better, it was a while ago, my gimp skills improved a lot since then.

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Well that's what I'm here for....I checked all the vehicles with both exe versions and have my findings on them listed at the bottom of the EXTRA check list above....forgot about blue team though with 64bit so I need to check that as well probably....if you do figure out how to fix that and it's something simple I would like to know so I can edit some character skins that have that issue, I know nothing about textures though so if it involves anything outside of Unreal Ed I probably won't be able to do it

@GreatEmerald 
Thanks to Alex I can use the Fury now and have edited the impact results with the Paladin hitting a Fury

**Comment by GreatEmerald** [comment]
Please keep posts in the correct issues, otherwise it gets difficult to track everything!

If you can figure out how the code could be changed to have the right scale for the parts without needing to change the animation files, that would be best. Otherwise I'd need to reupload those and that's a pain with GitHub. (I suppose I could put the animations/textures/etc. on NextCloud instead, but that would complicate the build process even further...) If it's not possible, then at least do all changes at once and upload the changed file only once.

If you have fixes pending, like with the lights, and want to submit them, please make a pull request. You need to fork this git repository, clone it, do your changes, commit them and upload to your branch, then make a pull request. I will then review it and accept it or request additional changes. That way we are not stuck with a bazillion different incompatible versions.

For the textures, make sure it works under 64-bit. It's better to lower the quality a bit to make it compatible than to have higher qualty that only works on 32-bit and looks bad on 64-bit. You might want to open a separate issue about the textures, since it's not specific to any one vehicle.

**Comment by Unre-Alex** [comment]
@HellDragon-HK Maybe create a new issue with all the light stuff, that way we will keep the things organized and won't bloat GE everywhere with it. Also, check your email.

@GreatEmerald How do you upload textures here, the way you posted a comment above? This place is a labyrinth.

**Comment by HellDragon-HK** [comment]
@Unre-Alex Got it, and sent you one back, I'll make a textures issue and move texture related entries to it.

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Hey did you get those 3 e-mails for each test I sent, I'm having net issues so I don't know if they sent or not

**Comment by HellDragon-HK** [comment]
Need some help checking this, for both the Paladin and Goliath in UT2004 if you start moving as soon as you get in it seems like the vehicle is at max speed instantly and doesn't need time to accelerate and for that matter I'm not sure the Paladin even needs time to accelerate from just normally stopping and moving again.

**Comment by GreatEmerald** [comment]
I tried to match the UT3 behaviour as much as I could, at least with the Goliath. It accelerates real fast in UT3 just as well. It's not instantaneous, though.

**Comment by HellDragon-HK** [comment]
Seems instant in UT2004 but like I said I need help confirming that.

**Comment by HellDragon-HK** [comment]
Updated the Paladin's collision and now it isn't being tossed around like it's weightless (or at least hasn't happened yet but it seems fixed)
https://drive.google.com/open?id=1c5lknHX3QvrfAccpvxneiM6f_6jf2Ipv

**Comment by HellDragon-HK** [comment]
Paladin needs a shield hit effect class like the Levi has because when it's blocking shots it uses the stock UT2004 shield and size

**Comment by HellDragon-HK** [comment]
Added a link for UT3 Drawscale ukx edit

**Comment by HellDragon-HK** [comment]
Forgot about Z Translation.....but doesn't seem to work on the Paladin, it's wheels keep sinking into the ground until it's all the way back to 1.0, any idea why, I assume it's something in the defaultproperties vehicle mass maybe?

**Comment by HellDragon-HK** [comment]
Ok changing the suspension bone offset fixed the Paladin and it is right with the UT3 drawscale on it's wheel size by the look of it with character measurement but the body isn't as high in UT3 so it must also be in a compressed state then like the Hellbender only this one stays that way I think since it doesn't have a rise animation

**Comment by HellDragon-HK** [comment]
Just noticed that the Paladin run over death message is run over by Scorpion, is that a mistake I've done or a bug?

**Comment by HellDragon-HK** [comment]
@GreatEmerald Ok cut it down into smaller issues, did I miss any?

**Comment by GreatEmerald** [comment]
Is the wheel plate issue different from the fender turning issue?

That's weird; `DamTypeRoadkill` seems to construct the name of the vehicle that ran over the enemy automatically. Something to look into perhaps.

**Comment by HellDragon-HK** [comment]
It's probably one and the same so I just missed removing it

class DamTypeRoadkill extends VehicleDamageType
	abstract;

var int MessageSwitchBase, NumMessages;

static function string DeathMessage(PlayerReplicationInfo Killer, PlayerReplicationInfo Victim)
{
	if (default.VehicleClass != None)
		return Default.DeathString @ default.VehicleClass.default.VehiclePositionString $ ".";
	else
		return Default.DeathString $ ".";
}

Each vehicle has it's own roadkill type, just DamTypeRoadkill is a base like ONSVehicle or TreadCraft, WheeledCraft, ect..   I think, I'm no expert here

**Comment by HellDragon-HK** [comment]
UKX updated to scale the cannon to the right size of 1.35 now that the default 0.8 which was making it small is overriden

**Comment by GreatEmerald** [comment]
Yea, and that code appends the name of the vehicle, because for instance for the Paladin:
```UnrealScript
VehiclePositionString="in a Paladin"
```
Unless that is set to say "in a Scorpion" that should never happen.

**Comment by HellDragon-HK** [comment]
So then why is it?

**Comment by HellDragon-HK** [comment]
Paladin driver is getting hurt from explosions like tank shells, not sure that happens in UT3 and haven't had luck trying to find out
